### DISCUSSION TITLE
Disassociate a fork from its upstream repository

### DISCUSSION THREAD

**Discussion by gforcada** [community/community#45251] [Repositories] [answered]
### Select Topic Area

General

### Body

Hi! 👋🏾 😄 

I forked a repository from one [organization](https://github.com/zopefoundation/meta) to [another](https://github.com/plone/meta), but not to contribute back, but rather to start our own journey with it 🚂 

It's a rather small repository that has a few scripts that make sense on the original organization and that we want to tailor to our needs on the other organization where we forked it.

In this specific use case it would be great to have a way to disassociate the forked repository from its original source as we history of both repositories will not be merged back.

The situation now is a bit unfortunate as you keep getting reminders that the branches have diverged. Whenever one creates a PR you are sent to the original repository and as they already diverged enough that PRs on the new repository generate conflicts on the original one and that confuses collaborators...

Does anyone have a solution for this? 🤔 🍀

**Comment by jge162** [comment]
Create a new repository: You can create a new repository within your organization, and then copy all the files from the forked repository to the new repository. This will give you a clean history and you won't have to deal with any merge conflicts.

**Reply by gforcada** [reply]
Yes that's a good solution when one has _not yet_ forked the repository, but as it is already done, and collaborators have started making contributions, deleting the fork and creating a new repository will break the local clones, git history commits etc... 😕

**Reply by jge162** [reply]
ah thought it was recent. you can create a new branch on your forked repository and work on that branch, instead of the default branch. This will keep the commit history but also keeps the new changes separate from the original repository and avoid conflicts.
you can also use git filter-branch to remove all the commits from the forked repository that you don't want to keep.

**Comment by airtower-luna** [comment, marked as answer]
There's a section in the support form for that: https://support.github.com/request/fork

I haven't had a need to use it myself, but it should let you disconnect the fork, so it'll become a standalone repo. :slightly_smiling_face:

**Reply by gforcada** [reply]
Oh, cool, thanks I did not find that link during my searches 🤷🏾
